src/components/ChannelsList/constants.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Liberty Global B.V.
```

Comment:
Please update the "copyright" year to 2021 for all new files you add in this pull request

Reply:
updated "copyright" year 2020 to 2021

Reply:
> Also 2 runtime issues noticed:
> 
> 1. when opening tv channel list I cannot use keyboard "down" key. I first need to press "up" to select the first item in the channel list. Better is to pre-select the currently active channel and so the up/down keys can work as expected.
> 2. when pressing enter to activate view channel, the channel is changed as expected. Press Backspace to close the menu and you will see channel bar below with wrong channel. See movie I will send you

Answering to point 1:

This is the existing behavior, you can see in movies and apps screen. once you launch the movies screen or apps screen focus will not be there and on key up only you will be able to select the list and then you can navigate key down/up/left/right.